spec/indexers/hyrax/resource_indexer_spec.rb

```diff
@@ -3,6 +3,7 @@
 require 'hyrax/specs/shared_specs'
 
 RSpec.describe Hyrax::ResourceIndexer do
+  let(:resource) { Hyrax::Resource.new }
```

Comment:
This is added to clarify and ensure the proper behavior. Prior to this, we had misleading successful test.

Reply:
resource is now required